src/common/_modules/function_admin/main.tf

```diff
@@ -0,0 +1,65 @@
+resource "azurerm_resource_group" "function_admin_itn_rg" {
+  name     = "${var.project_itn}-funcadm-rg-01"
+  location = var.location_itn
+
+  tags = var.tags
+}
+
+module "function_admin_dx" {
```

Comment:
if possible don't use `dx` in module name, use instead the region in case of duplication, it's more semantic